bin/hdfcoinc/pycbc_multiifo_coinc_findtrigs

```diff
@@ -0,0 +1,357 @@
+        ref = self.file['%s/%s_template' % (self.ifo, col)][num]
+        return self.file['%s/%s' % (self.ifo, col)][ref]
+
+    def set_template(self, num):
+        """ Set the active template to read from
+
+        Parameters
+        ----------
+        num: int
+            The template id to read triggers for
+
+        Returns
+        -------
+        trigger_id: numpy.ndarray
+            The indices of this templates triggers
+        """
+        self.template_num = num
+        times = self.get_data('end_time', num)
+
+        # Determine which of these template's triggers are kept after
+        # applying vetoes
+        if self.valid:
+            self.keep = veto.indices_within_times(times, self.valid[0], self.valid[1])
+            logging.info('applying vetoes')
+        else:
+            self.keep = numpy.arange(0, len(times))
+
+        if self.bank is not None:
+            self.param = {}
+            if 'parameters' in self.bank.attrs :
+                for col in self.bank.attrs['parameters']:
+                    self.param[col] = self.bank[col][self.template_num]
+            else :
+                for col in self.bank:
+                    self.param[col] = self.bank[col][self.template_num]
+
+        # Calculate the trigger id by adding the relative offset in self.keep
+        # to the absolute beginning index of this templates triggers stored
+        # in 'template_boundaries'
+        trigger_id = self.keep + self.file['%s/template_boundaries' % self.ifo][num]
+        return trigger_id
+
+    def __getitem__(self, col):
+        """ Return the column of data for current active template after
+        applying vetoes
+
+        Parameters
+        ----------
+        col: str
+            Name of column to read
+
+        Returns
+        -------
+        data: numpy.ndarray
+            The requested column of data
+        """
+        if self.template_num == None:
+            raise ValueError('You must call set_template to first pick the '
+                             'template to read data from')
+        data = self.get_data(col, self.template_num)
+        data = data[self.keep] if self.valid else data
+        return data
+
+logging.info('Starting...')
+
+num_templates = len(h5py.File(args.template_bank, "r")['template_hash'])
+tmin, tmax = parse_template_range(num_templates, args.template_fraction_range)
+logging.info('Analyzing template %s - %s' % (tmin, tmax-1))
+
+# Create dictionary for trigger files
+trigs={}
+for i in range(len(args.trigger_files)):
+    logging.info('Opening trigger file %s: %s' % (i,args.trigger_files[i]))
+    reader = ReadByTemplate(args.trigger_files[i],
+                                               args.template_bank,
+                                               args.segment_name,
+                                               args.veto_files)
+    trigs[reader.ifo] = reader
+
+# Set up pivot and fixed ifo for multi ifo coincidence
+triglist = list(trigs.values())
```

Comment:
This is the part  I mean. These lines aren't needed. Instead add options to the argument parser so that you can simply use args.pivot_ifo and args.fixed_ifo.

Reply:
@ahnitz ah ok, I misunderstood. We will have to set up these in the workflow generation than right, maybe even in the config files?

Reply:
Yes, that's right.

Reply:
We'll probably just do it in the config file if you are wondering.

Reply:
Yea, but if we do the detector pairs as well as the 3 detector, we will have to overwrite that for the detector pairs, that is why I was going with the input order previously. In the 2 detector case one is the pivot, the second is the fixed, and we can pick two in the 3 detector case, that was my thinking. But that overwriting can happen when setting up the coinc jobs in the workflow generation.